yada-api/src/main/resources/conf/YADA.properties

```diff
@@ -2,8 +2,6 @@
 YADA.server.http.port=${YADA.server.http.port}
 YADA.server.https.port=${YADA.server.https.port}
 YADA.server.context=${YADA.server.context}
-YADA.server.keystore.path=${YADA.server.keystore.path}
```

Comment:
I wonder if we could leave those statements there.
I think this keystore is in Dave's original repo because he needs to point to a TLS certificate, where we run with TLS in the container and leave TLS to Zephyr / or AWS ELB ...

Reply:
Hmm this is weird, whenever I view your comment from the "conversations" tab I see this as being deleted, however when I check the "Changed files" I see that the file was left intact. 

I had made both of this changes in my branch (fee2e48fabc087e9029a5d89f561a630d17646ff) already so I was expecting this to be resolved.

this is also happening for the .aspirin subgroups that you mentioned, is it possible that you saw an older commit diff? Or maybe I did something wrong on my end.